src/youtube/js/youtube.js

```diff
@@ -245,8 +246,13 @@ require("../../../html5-common/js/utils/constants.js");
      * @method OoyalaYoutubeVideoWrapper#play
      */
     this.play = function() {
-      if(!youtubePlayer) return;
-      if(playerReady)
+      if (youtubePlayer == null) {
```

Comment:
You check double equal null here but only check truthiness of youtubePlayer in onYouTubeIframeAPIReady. These should probably be consistent.